### ISSUE TITLE
Improve custom username character support

### ISSUE THREAD

**Issue by Lelahelry** [Omega-Numworks/Omega#196]
#### Describe the bug
When using an username that includes special characters, like "é", it doesn't display correcty in the settings menu, even though the rest of the settings actually support these characters.

#### Environment
 - Omega Version: 1.18.5 (or every other version)

**Comment by RedGl0w** [comment]
> When using an username that includes special characters, like "é", it doesn't display correctly in the settings menu, even though the rest of the settings actually support these characters.

Indeed, the settings can display accents : the most probably reason why it is not displaying for you may be that you're not using NKFD
